python/test/t/geometry/test_trianglemesh.py

```diff
@@ -287,6 +287,16 @@ def test_create_text():
     assert mesh.triangle.indices.shape == (936, 3)
 
 
+def test_create_from_volume():
+    coords = np.stack(np.meshgrid(*3 * [np.linspace(-1, 1, num=64)],
```

Comment:
```py
"""Create signed distance field for sphere of radius 0.5 and extract sphere from it."""
```

Reply:
Only needing a 3D Tensor or numpy array will make using this function straightforward. Maybe we can think of how to include VoxelGrid in another PR?

Reply:
Sounds good. The intention was that the current workflow creates a voxel grid through RGBD integration. We can add support for using this in RGBD integration later.